Libs/Visualization/VTK/Widgets/ctkVTKVolumePropertyWidget.cpp

```diff
@@ -219,17 +219,20 @@ void ctkVTKVolumePropertyWidget::updateFromVolumeProperty()
   vtkPiecewiseFunction* opacityFunction = 0;
   vtkPiecewiseFunction* gradientFunction = 0;
   bool scalarColorMapping = true;
-  if (d->VolumeProperty)
+  bool currentComponentValid = d->CurrentComponent >= 0 && d->CurrentComponent < VTK_MAX_VRCOMP;
+  this->setEnabled(d->VolumeProperty != nullptr && currentComponentValid);
```

Comment:
It is simple to disable the widget like this, but it may interfere with enabling/disabling of the widget by the parent widget. For example, the volume rendering widget may enable/disable the volume property widget based on some higher-level logic. So, I think we need to disable each child widget.

Reply:
Ok, I've disabled the child widgets.